src/12_scopes.py

```diff
@@ -16,9 +18,11 @@ def change_x():
 # This nested function has a similar problem.
 
 def outer():
+    global y
```

Comment:
Should use nonlocal for the second challenge, not global.

See: https://www.programiz.com/python-programming/global-local-nonlocal-variables